<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lukáš Tinkl &lt;ltinkl@redhat.com&gt;</author>
 <class>WifiSecurity</class>
 <widget class="QWidget" name="WifiSecurity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wifi Security</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="KComboBox" name="securityCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string comment="no security">None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WEP key 40/128 bits (Hex or ASCII)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WEP key 128 bits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LEAP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dynamic WEP (802.1x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WPA/WPA2 Personal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WPA/WPA2 Enterprise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WPA3 Personal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WPA3 Enterprise 192-bit</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Security:</string>
     </property>
     <property name="buddy">
      <cstring>securityCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="securityNone"/>
     <widget class="QWidget" name="securityWep">
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Key:</string>
         </property>
         <property name="buddy">
          <cstring>wepKey</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PasswordField" name="wepKey">
         <property name="maxLength">
          <number>26</number>
         </property>
         <property name="passwordModeEnabled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>WEP index:</string>
         </property>
         <property name="buddy">
          <cstring>wepIndex</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="wepIndex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>1 (default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Authentication:</string>
         </property>
         <property name="buddy">
          <cstring>wepAuth</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KComboBox" name="wepAuth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Open System</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shared Key</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="securityLeap">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Username:</string>
         </property>
         <property name="buddy">
          <cstring>leapUsername</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="leapUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="buddy">
          <cstring>leapPassword</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="PasswordField" name="leapPassword">
         <property name="passwordModeEnabled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="securityWpaPersonal">
      <layout class="QFormLayout" name="formLayout_4">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="buddy">
          <cstring>psk</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PasswordField" name="psk">
         <property name="passwordModeEnabled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>PasswordField</class>
   <extends>QLineEdit</extends>
   <header>passwordfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
